projects/plugins/crm/tests/php/automation/events/class-event-trigger-test.php

```diff
@@ -0,0 +1,92 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+
+namespace Automattic\Jetpack\CRM\Automation\Tests;
+
+use Automattic\Jetpack\CRM\Automation\Automation_Engine;
+use Automattic\Jetpack\CRM\Automation\Automation_Workflow;
+use Automattic\Jetpack\CRM\Automation\Triggers\Event_Deleted;
+use Automattic\Jetpack\CRM\Automation\Triggers\Event_New;
+use WorDBless\BaseTestCase;
+
+require_once __DIR__ . '../../tools/class-automation-faker.php';
+
+/**
+ * Test Automation's event triggers
+ *
+ * @covers Automattic\Jetpack\CRM\Automation
+ */
+class Event_Trigger_Test extends BaseTestCase {
+
+	private $automation_faker;
+
+	public function setUp(): void {
+		parent::setUp();
+		$this->automation_faker = Automation_Faker::instance();
+	}
+
+	/**
+	 * @testdox Test the event new trigger executes the workflow with an action
+	 */
+	public function test_event_new_trigger() {
+
+		$workflow_data = $this->automation_faker->workflow_without_initial_step_customize_trigger( 'jpcrm/event_new' );
+
+		$trigger = new Event_New();
```

Comment:
Just a suggestion that would increase my own for readability of the test, then this is never used until we run `init` further down, so that could be grouped together in the code

Reply:
Done: 08f1ff4